ui/components/app/edit-gas-fee-popover/network-statistics/tooltips.js

```diff
@@ -57,7 +57,7 @@ PriorityFeeTooltip.propTypes = {
   children: PropTypes.node.isRequired,
 };
 
-export const NetworkStabilityTooltip = ({ children, statusInfo }) => {
+export const NetworkStabilityTooltip = ({ children, color, tooltipLabel }) => {
```

Comment:
The tooltip doesn't really need the entirety of the status info object; all it cares about is the color and tooltip label.